go/cmd/server/main.go

```diff
@@ -41,7 +45,7 @@ func main() {
 	}
 
 	log.Println("Initializing feature store...")
-	fs, err := feast.NewFeatureStore(repoConfig)
+	fs, err := feast.NewFeatureStore(repoConfig, dummyTransformCallback)
```

Comment:
If we wanted to start the standalone go feature server, how can instantiate the transformation callback in python and pass it in here?

Reply:
Good question. With current implementation Go code must be always wrapped in Python, it's an easy way to have Python interpreter in the same process. Go feature server can run in standalone mode, but w/o support of transformations and python-based store connectors.